netlogo-gui/src/main/window/CodeEditor.scala

```diff
@@ -0,0 +1,18 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.window
+
+import javax.swing._
```

Comment:
I know I put this in here, let's see if we can import individual classes instead of a wildcard import.